hatch-fancy-pypi-readme is not available on Solaris.

Not intended for upstream.

--- twisted-25.5.0/pyproject.toml
+++ twisted-25.5.0/pyproject.toml
@@ -1,7 +1,6 @@
 [build-system]
 requires = [
     "hatchling >= 1.10.0",
-    "hatch-fancy-pypi-readme >= 22.5.0",
     "incremental >= 24.7.0",
 ]
 build-backend = "hatchling.build"
@@ -174,20 +173,6 @@ allow-direct-references = true
 [tool.hatch.version]
 source = "incremental"
 
-[tool.hatch.metadata.hooks.fancy-pypi-readme]
-content-type = "text/x-rst"
-fragments = [
-    { path = "README.rst" },
-]
-
-[[tool.hatch.metadata.hooks.fancy-pypi-readme.substitutions]]
-# Munge links of the form `NEWS <NEWS.rst>`_ to point at the appropriate
-# location on GitHub so that they function when the long description is
-# displayed on PyPI.
-pattern = '`([^`]+)\s+<(?!https?://)([^>]+)>`_'
-replacement = '`\1 <https://github.com/twisted/twisted/blob/trunk/\2>`_'
-ignore-case = true
-
 [tool.hatch.build.targets.wheel]
 packages = [
     "src/twisted",
